lib/sdl/Makefile

```diff
@@ -74,3 +74,30 @@ CLEANRULES += clean-libsdlimage
 .PHONY: clean-libsdlimage
 clean-libsdlimage:
 	$(VE)rm -f $(LIBSDLIMAGE_OBJS) $(NXDK_DIR)/lib/libSDL2_image.lib
+
+SDL_GFX_DIR = $(NXDK_DIR)/lib/sdl/SDL2_gfx
```

Comment:
SDL2_gfx and SDL_gfx are different projects which do the same thing, but have different dependencies and requirements.

It's realistic to imagine that someone will use some SDL 1.x implementation with nxdk.
In that case they might want to use SDL_gfx instead of SDL2_gfx.
Even if this is unlikely to happen in practice, we should still encourage a clean workspace and avoid such potential problems.

Hence we should ensure that we use distinct variable and filenames.